bucket/snaptimer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "0.1",
+    "description": "Free Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": "https://dan.hersam.com/software/license.php",
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": [
+        "md5:C145A92254BBFC5C08F23BD74AA5B073",
+        "sha1:7131F4C79038B8059F8EF16601077F075CFAD2DC"
+    ],
+    "extract_dir": "SnapTimer",
+    "shortcuts": [
+        [
+            "SnapTimer.exe",
+            "SnapTimer"
+        ]
+    ],
+    "persist": [
```

Comment:
- Do not use array for single item
- Does the file exist in the downloaded archive?